packages/asc-ui/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import TextArea from './TextArea'
+
+export default {
+  title: 'Experimental/Atoms/TextArea',
+
```

Comment:
Could you add a README with a link to the Design System here? You can copy the code from the [Accordion](https://github.com/Amsterdam/amsterdam-styled-components/blob/master/packages/asc-ui/src/components/Accordion/Accordion.stories.tsx#L11-L13) story as an example.

Just make sure to include the right link and mark the component as *experimental*.

Reply:
Done